frontend/test/metabase/scenarios/dashboard/actions-on-dashboards.cy.spec.js

```diff
@@ -99,6 +106,206 @@ const TEST_TABLE = "scoreboard_actions";
           expect(result.rows.length).to.equal(1);
         });
       });
+
+      it("adds an implicit create action to a dashboard and runs it", () => {
+        createModelFromTable(TEST_TABLE);
+        cy.get("@modelId").then(id => {
+          cy.request({
+            url: "/api/action",
+            method: "POST",
+            body: {
+              kind: "row/create",
+              name: "Create",
+              type: "implicit",
+              model_id: id,
+            },
+          });
+        });
+
+        cy.createDashboard({ name: `action packed dash` }).then(
+          ({ body: { id: dashboardId } }) => {
+            visitDashboard(dashboardId);
+          },
+        );
+
+        editDashboard();
+        cy.icon("click").click();
```

Comment:
Should we maybe add an `aria-label` to that button and switch to `findByLabelText`?